### ISSUE TITLE
[FEATURE REQUEST]: Radar Altimeter needs more screws for bezel/stepper mount stability

### ISSUE THREAD

**Issue by Higgins-OH** [jrsteensen/OpenHornet#1092]
### Discord Username

Higgins

### Feature/Enhancement Summary

The complete Radar Alt assembly is held by two screws only. 
To make things worse, these screws need to be installed quite loosely to reduce friction on the indexer ring.
If the knob is pushed (to test), it becomes obvious that a third screw is needed, because only gravity keeps the body in place. Preferrably at the bottom of the stepper mount.

### Screenshots/Images/Files

1: The only screws to connect bezel and body.
<img width="842" alt="image" src="https://github.com/user-attachments/assets/49c3da46-ff95-4ab1-bb53-def24692fd8b">

2: Effect of pushing knob when screws are loose enough for the indexer ring to move.

https://github.com/user-attachments/assets/0154e454-627a-4b8f-9f9c-8fbe6a5736db

### Justification

The whole assembly will be more rigid and at the same time allows the indexer ring to move.

### Applicable Part Numbers

OH5A2A4-1

### Release Version

v0.2.0

### Category

Mechanical (Structure/Panels/Mechanisms)

### Applicable End Item(s)

Right Console (RCON)

**Comment by kbastronomics** [comment]
i did a quick look and i do not see anywhere an additional mounting hole could be added and have enough integrity to be stable.  I tried a couple of things in fusion to add space but anytime I make the mounting hole the correct size it blows away a edge or something.

be interesting if anyone else can come up with a method

**Comment by Higgins-OH** [comment]
What if the stepper mount were a bit extended in one corner? It's similar to the upper screw and would definitely help.
The Off Flag wouldn't be affected.
<img width="668" alt="image" src="https://github.com/user-attachments/assets/a61d6627-f2d6-42d3-b3f5-f21f3acfb9d1">

**Comment by kbastronomics** [comment]
yes, that's what I tried,  there is only one spot I can see where a hole can be put that goes thru to the bezel and that spot it gets really thin on the bezel once a insert is put in place
